src/WingetCreateCLI/Commands/UpdateLocaleCommand.cs

```diff
@@ -0,0 +1,291 @@
+                            {
+                                NewLocaleCommand command = new NewLocaleCommand
+                                {
+                                    Id = this.Id,
+                                    Version = this.Version,
+                                    Locale = this.Locale,
+                                    GitHubToken = this.GitHubToken,
+                                };
+                                await command.LoadGitHubClient();
+                                Console.WriteLine();
+                                return await command.Execute();
+                            }
+                        }
+                    }
+                    catch (ArgumentException)
+                    {
+                        Logger.ErrorLocalized(nameof(Resources.InvalidLocale_ErrorMessage));
+                        return false;
+                    }
+                }
+
+                Manifests updatedLocales = this.PromptAndUpdateExistingLocales(originalManifests);
+                Console.WriteLine();
+                EnsureManifestVersionConsistency(originalManifests);
+                DisplayUpdatedLocales(updatedLocales);
+
+                if (string.IsNullOrEmpty(this.OutputDir))
+                {
+                    this.OutputDir = Directory.GetCurrentDirectory();
+                }
+
+                string manifestDirectoryPath = SaveManifestDirToLocalPath(originalManifests, this.OutputDir);
+
+                if (ValidateManifest(manifestDirectoryPath))
+                {
+                    if (Prompt.Confirm(Resources.ConfirmGitHubSubmitManifest_Message))
+                    {
+                        return await this.LoadGitHubClient(true) ?
+                            (commandEvent.IsSuccessful = await this.GitHubSubmitManifests(
+                                originalManifests,
+                                this.GetPRTitle(originalManifests, null, nameof(UpdateLocaleCommand))))
+                            : false;
+                    }
+                    else
+                    {
+                        Logger.WarnLocalized(nameof(Resources.SkippingPullRequest_Message));
+                    }
+                }
+                else
+                {
+                    return false;
+                }
+
+                return await Task.FromResult(commandEvent.IsSuccessful = true);
+            }
+            catch (Octokit.NotFoundException e)
+            {
+                Logger.ErrorLocalized(nameof(Resources.Error_Prefix), e.Message);
+                Logger.ErrorLocalized(nameof(Resources.OctokitNotFound_Error));
+                return false;
+            }
+            finally
+            {
+                TelemetryManager.Log.WriteEvent(commandEvent);
+            }
+        }
+
+        private static object PromptAllLocalesAsMenuSelection(Manifests originalManifests)
```

Comment:
This method should only take in what we need so the parameters should just be DefaultLocaleManifest and LocaleManifests.